pages/projects/submit.tsx

```diff
@@ -259,15 +272,22 @@ const SubmitProject = () => {
                           {...field}
                           multiple={false}
                           label="Drag or upload your Image."
-                          classes="file-uploader"
+                          classes={`file-uploader ${
+                            fileError && 'file-uploader-error'
+                          }`}
                           name={field.name}
+                          maxSize={2}
+                          types={fileTypes}
                           handleChange={(file: File) => {
+                            setFileError(false);
                             const reader = new FileReader();
                             reader.readAsDataURL(file);
                             reader.onload = (readerEvent) => {
                               field.onChange(readerEvent.target?.result);
                             };
                           }}
+                          onTypeError={() => setFileError(true)}
```

Comment:
`File type/size error, Hovered on types! JPG,PNG,GIF`  instead some like this 
<img width="296" alt="image" src="https://user-images.githubusercontent.com/60838584/193440730-d56d3d00-494c-49d3-b7b1-3f6491e97715.png">

Reply:
This will need more effort, the existing library used for file upload is not matured enough to customize error message, Will look into what needs to be done

Reply:
@leewan-09 fixed color issue

Reply:
> This will need more effort, the existing library used for file upload is not matured enough to customize error message, Will look into what needs to be done

if that case keep it for now